layer-atlas/src/main/res/layout/netguru_priority_label.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:background="@drawable/netguru_rounded_label"
+    android:paddingBottom="4dp"
```

Comment:
Shouldn't be in dimens? Same for `textSize`

Reply:
👍